<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="8193" started-at="2022-05-04T15:47:16Z" finished-at="2022-05-04T15:47:24Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="8193" started-at="2022-05-04T15:47:16Z" finished-at="2022-05-04T15:47:24Z">
      <class name="testcases.TC006_MS_CreateAnAccountFailedTest">
        <test-method status="PASS" signature="init()[pri:0, instance:testcases.TC006_MS_CreateAnAccountFailedTest@44a664f2]" name="init" is-config="true" duration-ms="10307" started-at="2022-05-04T15:47:05Z" finished-at="2022-05-04T15:47:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="FAIL" signature="CreateAnAccountTest()[pri:0, instance:testcases.TC006_MS_CreateAnAccountFailedTest@44a664f2]" name="CreateAnAccountTest" duration-ms="6789" started-at="2022-05-04T15:47:16Z" finished-at="2022-05-04T15:47:23Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [An account using this email address has already been registered. Please enter a valid password or request a new one] but found [An account using this email address has already been registered. Please enter a valid password or request a new one.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [An account using this email address has already been registered. Please enter a valid password or request a new one] but found [An account using this email address has already been registered. Please enter a valid password or request a new one.]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at testcases.TC006_MS_CreateAnAccountFailedTest.CreateAnAccountTest(TC006_MS_CreateAnAccountFailedTest.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateAnAccountTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testcases.TC006_MS_CreateAnAccountFailedTest@44a664f2]" name="tearDown" is-config="true" duration-ms="2033" started-at="2022-05-04T15:47:24Z" finished-at="2022-05-04T15:47:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.TC006_MS_CreateAnAccountFailedTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
